packages/ra-ui-materialui/src/field/ReferenceField.tsx

```diff
@@ -237,9 +237,9 @@ ReferenceFieldView.propTypes = {
     translateChoice: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]),
 };
 
-type ReferenceFieldClassKey = 'link';
+export type ReferenceFieldClassKey = 'link';
```

Comment:
Not sure if previous comment & fix is applicable here as well

Reply:
yes, fixed, too